### PR TITLE
Msdk 779 network namespace

### PR DESCRIPTION
**Pull request by polymath-eric**
Creates a network namespace that contains all previous middleware
methods as well as some from root

- sdk.getLastestBlock -> sdk.network.getLatestBlock
- sdk.getNetworkVersion -> sdk.network.getVersion
- sdk.getNetworkProperties -> sdk.network.getProperties
- sdk.getTransactionFees -> sdk.network.getTransactionFees
- sdk.getTreasuryAccount -> sdk.network.getTreasuryAccount
- sdk.getTreasuryBalance -> sdk.network.getTreasuryBalance
- sdk.transferPolyx -> sdk.network.transferPolyx
- sdk.middleware.getEventByIndexedArgs -> sdk.network.getEventByIndexedArgs
- sdk.middleware.getEventsByIndexedArgs -> sdk.network.getEventsByIndexedArgs
- sdk.middleware.getTransactionByHash -> sdk.network.getTransactionByHash

BREAKING CHANGE: 🧨 Moved methods

### CODE REVIEW

**Comment by prashantasdeveloper** [comment]
LGTM

**Comment by polymath-eric** [comment]
@monitz87 Prashant gave an approval for this one.

**Review by prashantasdeveloper — APPROVED** [review]

File: src/Network.ts [hunk 1]
```diff
@@ -1,26 +1,131 @@
...
+      version: u32ToBigNumber(specVersion).toNumber(),
+    };
+  }
+
+  /**
+   * Retrieve the protocol fees associated with running a specific transaction
+   *
+   * @param args.tag - transaction tag (i.e. TxTags.asset.CreateAsset or "asset.createAsset")
+   */
+  public getTransactionFees(args: { tag: TxTag }): Promise<BigNumber> {
```
Comment (monitz87) [inline src/Network.ts:85]: I think we should go ahead and change this (and the underlying context method) to `getProtocolFees`. When fetching fees for a `PolymeshTransaction`, they are broken down between `gas` and `protocol`, so we'd be more consistent

**Review by monitz87 — CHANGES_REQUESTED** [review]

**Review by monitz87 — APPROVED** [review]

**Pull request merged**